kahuna/.htmllintrc

```diff
@@ -28,7 +28,7 @@
     // our identation is a bit more complex than this, so don't check it
     "indent-width": false,
     "label-req-for": true,
-    "line-end-style": "lf",
+    "line-end-style": false,
```

Comment:
I added this because git was auto CLRF-ing my files. I did some reading and apparently this is recommended if you have users on different systems.